storage/scylla_api.h

```diff
@@ -14,11 +14,50 @@ extern "C" {
 #include "accelerator/errors.h"
 #include "cassandra.h"
 #include "common/model/transaction.h"
+#include "scylla_table.h"
 #include "utils/containers/hash/hash243_queue.h"
 #include "utils/logger.h"
 
+typedef struct {
+  CassCluster* cluster;
+  CassSession* session;
+} db_client_service_t;
+
 typedef struct scylla_iota_transaction_s scylla_iota_transaction_t;
 
+/**
+ * @brief init Scylla client serivce and connect to specific cluster
+ *
+ * @param[out] service Scylla client service
+ * @param[in] hosts Scylla cluster entry point IP address
+ * @return
+ * - SC_OK on success
+ * - non-zero on error
+ */
+status_t db_client_service_init(db_client_service_t* service, const char* hosts);
+
+/**
+ * @brief free Scylla client serivce
+ *
+ * @param[in] service Scylla client service
+ * @return
+ * - SC_OK on success
+ * - non-zero on error
+ */
+status_t db_client_service_free(db_client_service_t* service);
+
+/**
+ * @brief generate a new db_identity_t obj with specific hash and status
+ *
+ * @param[in] service Scylla client service
+ * @param[in] hash transaction hash to be set
+ * @param[in] status transaction status to be set
+ * @return
+ * - SC_OK on success
+ * - non-zero on error
+ */
+status_t db_insert_tx_into_idtable(db_client_service_t* service, int64_t ID, const char* hash, db_txn_status_t status);
```

Comment:
How about changing the function name to `db_insert_txn_into_identity`?

Reply:
OK!